resources/config/routing.yml

```diff
@@ -258,3 +257,13 @@ privacy:
    methods: [GET]
    controller: OpenCFP\Http\Action\Signup\PrivacyAction
 
+central:
+  path: /central
```

Comment:
When navigating to `/central`, after a redirect I end up here:

<img width="1792" alt="screen shot 2018-10-03 at 10 24 43" src="https://user-images.githubusercontent.com/605483/46402046-9b9fdd80-c6f6-11e8-8af6-880d23fff4e9.png">

What am I missing here?

Also see the `opencfpcentral.authorizeUrl` parameter.

Reply:
I have updated all the config files to use the correct authorizeUrl value